bindings/python/template/module.html.jinja2

```diff
@@ -0,0 +1,13 @@
+{% extends "default/module.html.jinja2" %}
```

Comment:
Do we need license header for this file?

Reply:
Nice to have. Done.